--- a/third_party/llama.cpp/CMakeLists.txt
+++ b/third_party/llama.cpp/CMakeLists.txt
@@ -32,5 +32,5 @@
 endif()
 
-option(LLAMA_USE_SYSTEM_GGML "Use system libggml" OFF)
+option(LLAMA_USE_SYSTEM_GGML "Use system libggml" ON)
 
 option(LLAMA_WASM_MEM64 "llama: use 64-bit memory in WASM builds" ON)
--- a/third_party/llama.cpp/ggml/CMakeLists.txt
+++ b/third_party/llama.cpp/ggml/CMakeLists.txt
@@ -175,6 +175,4 @@
 option(GGML_LSX              "ggml: enable lsx"               ON)
 option(GGML_RVV              "ggml: enable rvv"               ON)
-option(GGML_RV_ZFH           "ggml: enable riscv zfh"         ON)
-option(GGML_RV_ZVFH          "ggml: enable riscv zvfh"        ON)
 option(GGML_RV_ZICBOP        "ggml: enable riscv zicbop"      ON)
 option(GGML_RV_ZIHINTPAUSE   "ggml: enable riscv zihintpause" ON)
